File: 2227-sum-of-subarray-ranges/2227-sum-of-subarray-ranges.cpp
class Solution {
public:
    long long subArrayRanges(vector<int>& nums) {
        long long  sum =0;
        int largest, smallest;
        for(int i=0;i<nums.size();i++)
        {
            largest = nums[i];
            smallest = nums[i];
            for(int j=i+1;j<nums.size();j++)
            {
                largest = max(largest,nums[j]);
                smallest = min(smallest , nums[j]);

                sum += (largest-smallest);
            }
        }
        return sum;
    }
};

File: 2227-sum-of-subarray-ranges/2227-sum-of-subarray-ranges.java
class Solution {
    public long subArrayRanges(int[] nums) {
       long sum = 0;
       int largest , smallest;
       for(int i=0;i<nums.length;i++)
       {
        largest = nums[i];
        smallest = nums[i];
        for(int j=i+1;j<nums.length;j++)
        {
            largest = Math.max(largest,nums[j]);
            smallest = Math.min(smallest,nums[j]);

            sum += (largest-smallest);
        }
       }
       return sum;
    }
}